File: windows/net/samples.vb/CameraToDVD/VideoInfoForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace CameraToDVD
    Partial Public Class VideoInfoForm
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Info As String

        Private Sub VideoInfoForm_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            txtInfo.Text = Info
        End Sub

        Private Sub VideoInfoForm_Shown(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Shown
            txtInfo.Select(Info.Length, 0)
        End Sub

        Private Sub VideoInfoForm_KeyDown(ByVal sender As Object, ByVal e As KeyEventArgs) Handles MyBase.KeyDown
            If e.KeyCode = Keys.Escape Then
                Close()
            End If
        End Sub
    End Class
End Namespace

File: windows/net/samples/CameraToDVD/VideoInfoForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CameraToDVD
{
    public partial class VideoInfoForm : Form
    {
        public VideoInfoForm()
        {
            InitializeComponent();
        }

        public string Info;

        private void VideoInfoForm_Load(object sender, EventArgs e)
        {
            txtInfo.Text = Info;
        }

        private void VideoInfoForm_Shown(object sender, EventArgs e)
        {
            txtInfo.Select(Info.Length, 0);
        }

        private void VideoInfoForm_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Escape)
            {
                Close();
            }
        }
    }
}
